documents/transfer.md

```diff
@@ -233,6 +233,71 @@ List<Payment> payments = instance.payments.fetchAllTransfers(paymentId)
 ```
 -------------------------------------------------------------------------------------
 
+### Fetch transfer for an order
+
+```java
+String orderId = "order_JkaIDdkgGXVcwS";
+
+JSONObject request = new JSONObject();
+request.put("expand[]","transfers");
+
+Order response = razorpayclient.orders.get("orders/"+orderId,request);
```

Comment:
package razorpayclient does not exist

Reply:
made [correction](https://github.com/razorpay/razorpay-java/blob/transfer_doc_correction/documents/transfer.md?plain=1#L244)